client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -71,6 +71,8 @@ export default function IssueMultipleLicensesForm( {
 		[]
 	);
 
+	const licenseModalToBeShown = getQueryArg( window.location.href, 'show_license_modal' );
```

Comment:
In the pricing page, we used a hash fragment (e.g. https://cloud.jetpack.com/pricing#jetpack_starter_yearly)

 to determine which modal to display, as this usually meant we were trying to reach certain elements on the page.

What do you think of using hash fragment?

Reply:
I feel like it is better to use query parameters in this case. `hash` is mostly used when there is an identifier(id) set to an element. I also did some research, and it is recommended to use query parameters for the modal. So, I'm going to skip this.